ui/core/sim.ts

```diff
@@ -624,7 +624,12 @@ export class Sim {
 
 			const filters = proto.filters || Sim.defaultFilters();
 			if (filters.armorTypes.length == 0) {
-				filters.armorTypes = Sim.ALL_ARMOR_TYPES.slice();
+				// TODO: Can we specify the time of sim so that we can do this without querying the DOM?
+				if (!!document.querySelector('.not-within-raid-sim')) {
```

Comment:
Sadly not, because we don't have access to simUI here

Reply:
We could add it as a property when initializing and pass it when we initialize `raid_sim_ui`

Reply:
I would love for us to have a proper IsWithinRaidSim utility function in both the FE and BE code ^^